qc_fair_fairEva/tests/test_validator.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from types import SimpleNamespace
+
+from report2sqaaas_plugins_fairEva.main import fairEvaValidator
```

Comment:
tests are importing `fairEvaValidator` while the class name is `fairEva` in `qc_fair_fairEva/report2sqaaas_plugins_fairEva/main.py`